src/board/taxes/tax.js

```diff
@@ -1,17 +1,19 @@
-var Space = require('../board/space'), 
+var Space = require('../board/space'),
     inherits = require('util').inherits;
 
-function Tax() { 
-  this.taxAmount = 0; // give default value of 0, should be overridden 
+function Tax() {
+  this.taxAmount = 0; // give default value of 0, should be overridden
 }
 
 inherits(Tax, Space);
 
-Tax.prototype.performLandingAction = function(player) { 
+Tax.prototype.performLandingAction = function(game) {
   // should probably refactor this into a fn
   // that checks balance, prompts player to mortage/sell
   // before declaring him bankrupt
-  this.player.makePayment(this.amount);
-}; 
+  game.getCurrentPlayer().makePayment(this.taxAmount);
+  Tax.super_.prototype.performLandingAction.call(this, game);
```

Comment:
Thanks for adding all these super_ calls, was on my todo but hadn't gotten around to it